backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/MaskinportenSchemaController.cs

```diff
@@ -121,13 +117,13 @@ public async Task<ActionResult<List<DelegationsFE>>> GetAllOutboundDelegations([
         }
 
         /// <summary>
-        /// Endpoint for retrieving delegated resources between parties
+        /// Endpoint for revoking a maskinporten scope delegation on behalf of the party having received the delegation
         /// </summary>
         /// <response code="400">Bad Request</response>
         /// <response code="500">Internal Server Error</response>
         [HttpPost]
         [Authorize]
-        [Route("accessmanagement/api/v1/{party}/delegations/maskinportenschema/received/revoke")]
+        [Route("accessmanagement/api/v1/{party}/maskinportenschema/offered/revoke")]
```

Comment:
From the past code and the name of the method, i see that this must be <<received>> in the route instead of <<offered>>